package.json

```diff
@@ -114,7 +112,9 @@
     "@types/lodash": "4.14.194",
     "@types/node": "18.16.0",
     "@types/store": "2.0.2",
+    "@typescript-eslint/eslint-plugin": "^4.33.0",
     "babel-jest": "29.5.0",
+    "babel-eslint": "^10.1.0",
```

Comment:
eslintrc was using this already but we never explicitly installed it. Just happened to have it as a sub dependency.